CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ Package-specific changes not released in any SDK will be added here just before
 - Updated `@shopify/react-native-skia` from `0.1.157` to `0.1.171`. ([#20857](https://github.com/expo/expo/pull/20857) by [@kudo](https://github.com/kudo))
 - Updated `react-native-safe-area-context` from `4.4.1` to `4.5.0`. ([#20899](https://github.com/expo/expo/pull/20899) by [@gabrieldonadel](https://github.com/gabrieldonadel))
 - Updated `react-native-screens` from `3.18.0` to `3.19.0`. ([#20938](https://github.com/expo/expo/pull/20938) by [@lukmccall](https://github.com/lukmccall))
+- Updated `@react-native-community/slider` from `6.0.1` to `6.1.2`. ([#20932](https://github.com/expo/expo/pull/20932) by [@gabrieldonadel](https://github.com/gabrieldonadel))
```

Comment:
I believe there is a typo here, should have been `react-native-pager-view`

Reply:
Thanks for highlighting this @buraks, this has been fixed on https://github.com/expo/expo/commit/5cb3fbeaa9400947e854ff31ee078859347f7bbd